packages/nube-devtools/README.md

```
@@ -0,0 +1,34 @@
+## Installing
+
+1. Check if your `Node.js` version is >= **14**.
+2. Change or configurate the name of your extension on `src/manifest`.
+3. Run `npm install` to install the dependencies.
+
+## Developing
+
+run the command
+
+```shell
+$ cd nube-devtools
```

markdownlint-cli2 (0.17.2)

Dollar signs used before commands without showing output
null

(MD014, commands-show-output)